rpc/backend/backend_suite_test.go

```diff
@@ -49,7 +49,7 @@ func (suite *BackendTestSuite) SetupTest() {
 	ctx.Viper.Set("telemetry.global-labels", []interface{}{})
 
 	baseDir := suite.T().TempDir()
-	nodeDirName := "node"
+	nodeDirName := fmt.Sprintf("node")
```

Comment:
This seems unnecessary
```suggestion
	nodeDirName := "node"
```